hudi-client/hudi-client-common/src/test/java/org/apache/hudi/utils/HoodieWriterClientTestHarness.java

```diff
@@ -149,6 +149,8 @@ public abstract class HoodieWriterClientTestHarness extends HoodieCommonTestHarn
 
   protected abstract BaseHoodieWriteClient getHoodieWriteClient(HoodieWriteConfig cfg);
 
+  protected abstract BaseHoodieWriteClient getHoodieWriteClient(HoodieWriteConfig cfg, boolean shouldCloseOlderClient);
```

Comment:
The method implies the write client could be cached? Like through `getHoodieWriteClient(HoodieWriteConfig cfg)`